mne/preprocessing/infomax_.py

```diff
@@ -13,94 +13,92 @@
 
 @verbose
 def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
-            anneal_deg=60., anneal_step=0.9, extended=False, n_subgauss=1,
-            kurt_size=6000, ext_blocks=1, max_iter=200,
-            random_state=None, blowup=1e4, blowup_fac=0.5, n_small_angle=20,
-            use_bias=True, verbose=None):
-    """Run the (extended) Infomax ICA decomposition on raw data
+            anneal_deg=60., anneal_step=0.9, extended=True, n_subgauss=1,
+            kurt_size=6000, ext_blocks=1, max_iter=200, random_state=None,
+            blowup=1e4, blowup_fac=0.5, n_small_angle=20, use_bias=True,
+            verbose=None):
+    """Run (extended) Infomax ICA decomposition on raw data.
 
-    based on the publications of Bell & Sejnowski 1995 (Infomax)
-    and Lee, Girolami & Sejnowski, 1999 (extended Infomax)
+    Based on the publications of Bell & Sejnowski, 1995 (Infomax)
+    and Lee, Girolami & Sejnowski, 1999 (extended Infomax).
```

Comment:
while you're at it, could you include the full references to these papers in a `References` section of the docstring?